azure-communication-ui/call-with-chat/src/main/java/com/azure/android/communication/ui/callwithchat/CallWithChatManager.kt

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.callwithchat
+
+import android.content.Context
+import com.azure.android.communication.ui.calling.CallComposite
+import com.azure.android.communication.ui.callwithchat.models.CallWithChatCompositeLocalOptions
+import com.azure.android.communication.ui.callwithchat.models.CallWithChatCompositeRemoteOptions
+import com.azure.android.communication.ui.chat.ChatComposite
+// import com.azure.android.communication.ui.chat.implementation.ChatServiceConfigurationImpl
```

Comment:
Should delete comments. If you need reference of source Private repo still will exist.